apps/frontend/.env

```diff
@@ -0,0 +1 @@
+BACKEND_URL=localhost:8000
```

Comment:
hatte ich in der .env auf 8101 geändert, müssen mal schauen ob das so passt

Reply:
erledigt